quantum/process_keycode/process_unicode_common.c

```diff
@@ -20,6 +20,7 @@
 #include <ctype.h>
 
 static uint8_t input_mode;
+static bool first_flag = false;
```

Comment:
This can be made a static local variable inside `prepare_unicode_input_mode` since it isn't needed anywhere else.